roles/frontend/templates/nginx-default.d-webroot.conf.j2

```diff
@@ -1,9 +1,32 @@
 location / {
+    try_files $uri @prerender;
+}
+
+location @prerender {
+
     proxy_cache frontend_cache;
     proxy_cache_valid 200 302 10m;
     proxy_cache_valid 301      1h;
     proxy_cache_valid any      1m;
-    proxy_pass {{ frontend_nginx_webroot_origin }};
+
+    proxy_buffering on;
+    proxy_cache_revalidate on;
+    proxy_cache_min_uses 1;
+    proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
```

Comment:
Can remove, is already present in templates/nginx-conf.d-cache.conf.j2

Reply:
Done